src/app/user/login/index.njk

```diff
@@ -47,9 +46,9 @@
 			<p class="govuk-body"><a href="/user/register">Create an account</a></p>
 
 			<button type="submit" class="govuk-button">
-				Agree and submit
+				Sign in
```

Comment:
Sign in button should be above the create an account section according to screenshot / prototype

Reply:
That was a big miss!- Must have gone screen blind. Because my commits weren't being signed I had to squash my commits and force push (with the help of @pawniemiec ). So hopefully this and the heading styling should be fixed.